filter-assingment/Akshay/filter-products/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link
+      href="https://unpkg.com/tailwindcss@^1.0/dist/tailwind.min.css"
+      rel="stylesheet"
+    />
+  </head>
+  <body class="bg-gray-100">
+    <header class="text-center p-8">
+      <h1
+        class="text-4xl font-semibold text-pink-600"
+        style="font-family: sail;"
+      >
+        Our Patisserie
+      </h1>
+    </header>
+    <section
+      class="flex justify-between max-w-screen-xl mx-4 my-8 md:mx-auto flex-col md:flex-row"
+    >
+      <div class="md:mb-0 mb-4">
+        <button
+          class="bg-white hover:bg-gray-100 text-gray-800 font-semibold py-2 px-4 border border-gray-400 rounded-lg mr-2 filter-button"
+          onclick="filterByType('all')"
```

Comment:
It will work but just good practice. 

It would have been good, If we use id as click param. 
`onclick="flterByType(this)"`  and then we listen `e.target.id ` Hard coded values can change and we will need to modify all the places of filters. 

In usual scenario we generate filter/ search params through dynamic array. So just good practice.